### ISSUE TITLE
shebang command not found

### ISSUE THREAD

**Issue by alvinj** [VirtusLab/scala-cli#432]
Using `scala-cli` version 0.0.8 on macOS, I get a `shebang: not found` error when trying to run the `shebang` examples on [scala-cli.virtuslab.org/docs/guides/scripts](https://scala-cli.virtuslab.org/docs/guides/scripts). This is what I get when I run the command at the command line:

```sh
$ scala-cli shebang
shebang: not found
```

I’m not sure if I’m reading that document correctly, but that’s what I get. (I notice that 'shebang' is not in the help text, so maybe I’m doing something wrong?)

**Comment by alvinj** [comment]
After further testing, I see that the script approach works with just this header:

```sh
#!/usr/bin/env -S scala-cli
```

So now I’m guessing that the `shebang` command isn’t required any more?

**Comment by alexarchambault** [comment]
It is actually required, it's just not in the current release (it'll be in the upcoming one). Ideally, it should have appeared on the website only after the next release. We're working on a way to ensure new features only land in the documentation upon release to prevent that kind of thing from happening.

**Comment by romanowski** [comment]
The first approach to update website with the release: https://github.com/VirtusLab/scala-cli/pull/433

**Comment by romanowski** [comment]
`shebang` is present in freshly released 0.0.9 version
